### PR TITLE
Updated images in styleguide for sub brands, bumped version

### PR DESCRIPTION
**Pull request by nmcclay**

### CODE REVIEW

File: app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu-product.scss [hunk 1]
```diff
@@ -31,12 +35,33 @@
...
 }
 .dropdown-menu-product-img {
-  padding-top: 10px;
-  max-height: 60px;
+  max-height: $product-image-base-height;
+
+  @media(max-width: $screen-sm) {
+    max-height: $product-image-base-height * $product-image-lg-factor;
+  }
+  @media(max-width: $screen-lg ) and (min-width: $screen-sm) {
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu-product.scss]: whitespace

File: app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu-product.scss [hunk 2]
```diff
@@ -31,12 +35,33 @@
...
 .dropdown-menu-product-item {
   margin-bottom: 30px;
-  height: 60px;
+  height: $product-image-base-height;
   text-align: center;
+
+  @media(max-width: $screen-sm) {
+    height: $product-image-base-height * $product-image-lg-factor;
+  }
+  @media(max-width: $screen-lg ) and (min-width: $screen-sm) {
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu-product.scss]: whitespace

**Comment by schweitzdesign** [comment]
Updated images look sharp.

There is a list item length issue at between 768px and 938px wide:
screen shot 2015-08-18 at 8 39 15 am

**Comment by schweitzdesign** [comment]
There's a leftover blue border on the active item in the :focus state

screen shot 2015-08-20 at 12 50 44 pm

File: app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss [hunk 3]
```diff
@@ -24,7 +32,11 @@
     padding: 10px 30px;
   }
 
-  @media(max-width: $screen-sm) {
+  @media(max-width: $grid-float-breakpoint) {
+    position: relative;
+    top: 0;
+    width: 100%;
+    //display: none;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss]: Can remove this commented out line.

File: app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss [hunk 4]
```diff
@@ -24,7 +32,11 @@
     padding: 10px 30px;
   }
 
-  @media(max-width: $screen-sm) {
+  @media(max-width: $grid-float-breakpoint) {
+    position: relative;
+    top: 0;
+    width: 100%;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss]: I don't think this is necessary.

File: app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss (hunk 3 above)
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss]: Actually, now I see that being necessary at the mid-size break over.

**Comment by schweitzdesign** [comment]
Active item causes some issues at mobile breakpoints as well.


screen shot 2015-08-20 at 2 20 09 pm

**Comment by schweitzdesign** [comment]
Effect and mobile implmentation look really good, just a few notes on active state.

File: app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss (hunk 3 above)
Comment (nmcclay) [inline app/assets/stylesheets/forever_style_guide/modules/_dropdown-menu.scss]: Fixed, thanks

**Comment by nmcclay** [comment]
I found one more tiny focus related bug... just a minute.

**Comment by schweitzdesign** [comment]
Adding the actual descriptions in there really brings this to life. Looks good!

**Pull request merged**